### PR TITLE
fix: wrapping catalog search results in paragon container to fit page formatting

### PR DESCRIPTION
**Pull request by alex-sheehan-edx**
With

image

Without

image

# For all changes

- [ ] Ensure adequate tests are in place (or reviewed existing tests cover changes)

# Only if submitting a visual change

- [ ] Ensure to attach screenshots
- [ ] Ensure to have UX team confirm screenshots

### CODE REVIEW

File: src/components/catalogPage/CatalogPage.jsx [hunk 1]
```diff
@@ -139,7 +140,9 @@ const CatalogPage = ({ intl }) => {
             messages['catalogPage.catalogSelectionDeck.title'],
           )}
         />
-        <CatalogSearch />
+        <Container size="xl">
+          <CatalogSearch />
```
Comment (adamstankiewicz) [inline src/components/catalogPage/CatalogPage.jsx]: While this works, you'll technically rendering a `Container` in a `Container` which in theory could lead to some unwanted behavior down the road.

Is there a way to update the `size` prop on the `Container` that already renders in `CatalogSearch` via its use of `PageWrapper`?

Comment (adamstankiewicz) [reply src/components/catalogPage/CatalogPage.jsx]: https://github.com/openedx/frontend-app-enterprise-public-catalog/blob/de642b2bc604cd2ed656ab9c688f398cdab7e96a/src/components/PageWrapper.jsx#L11

Comment (alex-sheehan-edx) [reply src/components/catalogPage/CatalogPage.jsx]: 👍

File: src/components/PageWrapper.jsx [hunk 2]
```diff
@@ -7,15 +7,16 @@ import { PAGE_TITLE } from '../constants';
...
+const PageWrapper = ({ children, className, size }) => (
+  <Container className={className} size={size}>
     <Helmet title={PAGE_TITLE} />
     <div data-testid={DATA_TEST_ID}>{children}</div>
   </Container>
 );
 
 PageWrapper.defaultProps = {
   className: '',
+  size: '',
```
Comment (adamstankiewicz) [inline src/components/PageWrapper.jsx]: I believe using empty string for prop to `Container` would result in a prop type warning:

`size` only accepts these types: https://github.com/openedx/paragon/blob/498357621207286f693bafedd6edf29762448dad/src/Container/index.jsx#L36

Use `undefined` here to match what `Container` expects.

**Review by adamstankiewicz — APPROVED** [review]

File: src/components/PageWrapper.jsx (hunk 2 above)
Comment (alex-sheehan-edx) [reply src/components/PageWrapper.jsx]: gooood catch

**Pull request merged**